packages/application-shell/src/apollo-links/header-link.js

```diff
@@ -7,7 +8,9 @@ import {
   selectUserId,
 } from '../utils';
 
-const isKnownTarget = target => Object.values(GRAPHQL_TARGETS).includes(target);
+const isKnownGraphQlTarget = target =>
+  Object.values(GRAPHQL_TARGETS).includes(target);
+const omitEmptyHeaders = headers => omitEmpty(headers);
```

Comment:
Nit: is this function necessary? Why not simply using `omitEmpty` directly?

Reply:
> Just to give it a "meaningful" name.

To be honest, I find `headers: omitEmpty({ ... })` already quite clear. But yeah, was just curious